marimo/_plugins/ui/_impl/input.py

```diff
@@ -562,13 +517,22 @@ def __init__(
                 raise ValueError("A radio group cannot have repeated options.")
             options = {option: option for option in options}
         self.options = options
+        if not inline and chunk is not None:
+            chunk = None
+        if (
+            inline
```

Comment:
what happens if it's out of bounds? If nothing happens on the frontend, it's probably ok to still pass it for correctness